test/test_checkers.py

```diff
@@ -117,3 +117,38 @@ def test_filename_is(self, checker_name, file_name, expected_results):
 
                 for result, expected_result in zip(results, expected_results):
                     assert result["is_or_contains"] == "is"
+
+    def test_glibc_latest_version(self):
+        """Checker didn't detect latest version of glibc"""
+
+        common_versions = [
+            # common strings generated by a compiler
+            r"$Pj?PW\n"
+            r"D$ P\n"
+            r"D$0PW\n"
+            r"GNU C Library (GNU libc) release release version\n",
+            r"Mandatory or optional arguments to long options are also mandatory or optional for any corresponding short options.\n",
+            r"Compiled by GNU CC version 10.2.1 20200804 (Red Hat 10.2.1-2).\n",
+            # bare version strings.
+            r"GLIBC_2.0\n",
+            r"GLIBC_2.1\n",
+            r"GLIBC_2.1.1\n",
+            r"GLIBC_2.3\n",
+            r"GLIBC_2.3.2\n",
+            r"GLIBC_2.12\n",
+            r"GLIBC_2.23\n",
+            r"GLIBC_2.30\n",
+            r"GLIBC_2.32\n",
+        ]
+        file_name = "libc.so.6"
+        checkers = pkg_resources.iter_entry_points("cve_bin_tool.checker")
+        result = None
+        for checker in checkers:
+            if checker.name == "glibc":
+                Checker = checker.load()
+                checker = Checker()
+
+                result = checker.get_version(common_versions, file_name)
+
+        if "is_or_contains" in result:
+            assert result["version"] == "2.32"
```

Comment:
We also are going to want something that asserts that we didn't find some of the other potential versions.  
```suggestion
            assert result["version"] == "2.32"
            assert "2.1" is not in result["version"] 
```

or somesuch (my syntax may be off).  I'd suggest you put negative asserts in for all of the options that are not what you're looking for here.

Reply:
Yeah. Negative asserts will make much more sense

Reply:
It will be ```assert "2.1" != "2.2"``` result['version'] will be a string here and not a list